nginx-custom.conf

```diff
@@ -5,5 +5,7 @@ server {
    index docs/next/overview.html docs/next/overview.html;
    try_files $uri $uri/ /docs/next/$request_uri =404;
    error_page 403 404 /docs/next/overview.html;
+   expires 2d;
+   add_header Cache-Control "public, no-transform";
```

Comment:
1. The `no-transform` thing is great but it might not play well with our end-users on slow networks. 
Ref: [Google WebLight](https://stackoverflow.com/questions/20134257/any-reason-not-to-add-cache-control-no-transform-header-to-every-page).
Ref: https://tools.ietf.org/html/rfc7234#section-5.2.2.4
It is a good thing but might hamper end-users' navigation.
I'd let the team decide what's best on slow networks, **familiar navigation** or **speed**.
cc @ranjithwingrider @visualapps 

Ref: [Test results](http://www.webpagetest.org/result/190811_HK_fce13234b62fa2fe353d235a23d1a7e9/)

```suggestion
   add_header "no-cache";
```
2. `no-cache` should be used so that older response is never used to satisfy a latter request without successful validation on the origin server, this will ensure that the changes are visible right away whenever they are requested again(without a hard reload).

3. `public` seems a bit unnecessary. Ref: [RFC7234](https://tools.ietf.org/html/rfc7234#section-5.2.2.5)

I was also going to suggest, `max-age` with `must-revalidate` but after going through this [SO Q&A](https://stackoverflow.com/questions/18148884/difference-between-no-cache-and-must-revalidate) and green check marks from REDbot, I'm okay with just `no-cache` which should ensure that older responses are never served.

Reply:
cool so I have added `add_header Cache-Control "no-cache, max-age=600";` as requested.